app/src/main/java/com/kickstarter/ui/compose/designsystem/KSButtons.kt

```diff
@@ -266,18 +263,16 @@ fun KSPrimaryGreenButton(
     textStyle: TextStyle = typographyV2.body,
     isEnabled: Boolean
 ) {
-    CompositionLocalProvider(LocalRippleTheme provides KSRippleThemeWhite) {
```

Comment:
`KSRippleTheme*` were used only for the Design System activity to provide the ripple animation. Providing the ripple animation was the only real conflict unable to co-exist M1.6 and M3 versions.

With M3 ripple can be done in **any** composable via `Modifier.clickable` :
```

Box(
    Modifier
        .clickable(
            interactionSource = remember { MutableInteractionSource() },
            indication = rememberRipple(bounded = true, radius = 24.dp)
        )
) {
    Text("Custom clickable with ripple")
}


```